proposal/09510/README.rst

```diff
@@ -0,0 +1,64 @@
+========================
+Encryption Acceleration 
+========================
+
+
+Basic Information
+========================
+Code repository: https://github.com/09510/Encryption-Acceleration
+
+Problem to solve
+=======================
+Message encryption is an important technology to protect information security. 
+However, if you want to ensure the encryption strength, the encryption/decryption takes a lot of time.
+Encryption Acceleration is a tool to accelerate encryption and decryption by GPU. 
+
+
+
+Perspective users 
+=========================
+Anyone who wants to use GPU to accelerate encryption and decryption.
+
+
+ 
+System architecture 
+=============================
+1. Calling encrypt/decrypt function by Python.
+2. Doing encrypt/decrypt by gpu.
+3. Return encrypted/decrypted msg. 
+    
+
+
+API description 
+================================
+Python API : 
```

Comment:
This may be too trivial to earn a decent grade.  The Python standard library already provides such API and you will need to prove that your approach differs from it and is more useful.